pkg/scraper/jobcenter_scraper.go

```diff
@@ -41,13 +70,22 @@ func ScrapeJobcenter() []model.Job {
 	// Limit to two pages
 	for i := 1; i < 3; i++ {
 
-		url := fmt.Sprintf("https://www.jobcentrebrunei.gov.bn/web/guest/search-job?q=&delta=200&start=%s", strconv.Itoa(i))
+		url := fmt.Sprintf("https://www.jobcentrebrunei.gov.bn/web/guest/search-job?q=&delta=%s&start=%s", strconv.Itoa(delta), strconv.Itoa(i))
 
-		if err := c.Visit(url); err != nil {
+		if err := linksCollector.Visit(url); err != nil {
 			fmt.Println("Error: ", err)
 			break
 		}
 	}
 
+	for i := range jobs {
+		jobs[i].Description = descriptions[i]
+	}
+
 	return jobs
 }
+
+func getJobcenterJobId(s string) string {
+	r := regexp.MustCompile(`^\/web\/guest\/view-job\/-\/jobs\/(?P<jobId>\d+)\/.*$`)
+	return fmt.Sprintf("jobcenter-%s", r.FindStringSubmatch(s)[1])
```

Comment:
I think instead of making the key unique this way, we can make the database table's provider id and provider job id a unique composite key?
Then if we insert a combination for jobcenter and the job id that is not unique, it will throw an error.

Reply:
What happens if it doesn't find a match?

Would it not turn into a nil memory error?

Should check and return err if that's the case.